From 7ed0e7da29046f2c1a6dd96fe7b1b34fafa06494 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Julian=20R=C3=BCth?= <julian.rueth@fsfe.org>
Date: Tue, 20 Jun 2023 16:04:03 +0300
Subject: [PATCH 09/11] Fix order of clang libraries

clangFrontend needs symbols from clangDriver so it needs to come first
---
 src/interpreter/cling/lib/Interpreter/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/interpreter/cling/lib/Interpreter/CMakeLists.txt b/src/interpreter/cling/lib/Interpreter/CMakeLists.txt
index 87e8353..841f68b 100644
--- a/src/interpreter/cling/lib/Interpreter/CMakeLists.txt
+++ b/src/interpreter/cling/lib/Interpreter/CMakeLists.txt
@@ -9,8 +9,8 @@
 set(LIBS
   clingUtils
   clangCodeGen
-  clangDriver
   clangFrontend
+  clangDriver
   clangParse
   clangSema
   clangAnalysis
-- 
2.46.0

